lib/compiler/graph-builder.js

```diff
@@ -173,10 +174,23 @@ class GraphBuilder {
         }
         return key + '[' + this.input_seqnos[key]  + ']';
     }
-    define_input(bname, input_name, options, isStatic) {
-        var input = {id: bname, name: input_name, options: options, static: isStatic};
-        input.value = this._compute_input_val(input);
```

Comment:
The `this._compute_input_val` method is no longer used, it should be deleted together with the “input value” section of `test/compiler/inputs.spec.js`.